apps/docs-snippets/src/guide/cookbook/custom-transactions-contract-calls.test.ts

```diff
@@ -63,7 +63,12 @@ describe('Custom Transactions from Contract Calls', () => {
     const response = await senderWallet.sendTransaction(transactionRequest);
     await response.waitForResult();
     // Get result of contract call
-    const { value } = await FunctionInvocationResult.build([scope], response, false, contract);
+    const { value } = await buildSubmitResult({
```

Comment:
Given `submit` is just a provider operation, should this be `buildFunctionResult`?

Reply:
https://github.com/FuelLabs/fuels-ts/pull/2652/commits/4b73be269c240f4647a3977fe9504a067341341c